### PR TITLE
Exp/alpha

### PR DESCRIPTION
**Pull request by choufractal**

### CODE REVIEW

File: automatic_data_generation/data/base_dataset.py [hunk 1]
```diff
@@ -254,25 +250,20 @@ def select_none_intents(self, dataset_folder, restrict_intents,
...
+                if restrict_intents is not None and intent not in restrict_intents:
                     continue
                 if cosine(none_vector, intent_vector) > cosine_threshold:
-                    LOGGER.info('none intent {} is close to {}'.format(
-                        none_intent, intent))
+                    print('none intent {} is close to {}'.format(none_intent, intent))
                     selected_none_intents.append(none_intent)
+                    pseudolabels[none_intent] = intent
                     break
+        return selected_none_intents, pseudolabels
```
Comment (choufractal) [inline automatic_data_generation/data/base_dataset.py:282]: `selected_none_intents` and `pseudolabels` are not defined

File: automatic_data_generation/data/base_dataset.py [hunk 2]
```diff
@@ -212,19 +213,14 @@ def build_data_files(self, dataset_folder, dataset_size=None,
...
-            new_train = self.add_nones(
-                new_train, none_folder, none_size=none_size,
-                none_intents=none_intents, none_idx=none_idx
-            )
-            new_test = self.add_nones(
-                new_test, none_folder, none_size=200,
-                none_intents=none_intents, none_idx=none_idx
-            )
+            pseudolabels = None
+            if infersent_selection is not 'no_infersent_selection':
```
Comment (choufractal) [inline automatic_data_generation/data/base_dataset.py]: better to replace `is not` by `!=`

File: automatic_data_generation/data/handlers/snips_dataset.py [hunk 3]
```diff
@@ -84,7 +85,10 @@ def add_nones(self, sentences, none_folder, none_size=None,
         for row in none_sentences[:none_size]:
             if 'snips' in str(none_folder):
                 new_row = row
-                new_row[3] = 'None'
+                if pseudolabels:
```
Comment (choufractal) [inline automatic_data_generation/data/handlers/snips_dataset.py]:
```
if pseudolabels is not None:
```

File: automatic_data_generation/data/base_dataset.py (hunk 2 above)
Comment (choufractal) [reply automatic_data_generation/data/base_dataset.py]: also, you should use the constant `NO_INFERSENT_SELECTION`

**Review by francescocalta — APPROVED** [review]

**Pull request merged**